tests/test_skip_pg_hba.py

```diff
@@ -0,0 +1,204 @@
+import pgautofailover_utils as pgautofailover
+from nose.tools import *
+
+import subprocess
+import os, os.path, time
+
+cluster = None
+node1 = None
+node2 = None
+
+def setup_module():
+    global cluster
+    cluster = pgautofailover.Cluster()
+
+def teardown_module():
+    cluster.destroy()
+
+def test_000_create_monitor():
+    monitor = cluster.create_monitor("/tmp/skip/monitor",
+                                     authMethod="skip",
+                                     sslMode="verify-ca")
+    monitor.wait_until_pg_is_running()
+
+    with open(os.path.join("/tmp/skip/monitor", "pg_hba.conf"), 'a') as hba:
+        hba.write("hostssl all all 172.27.1.0/24 cert\n")
```

Comment:
consider defining the network prefix in pautofailover_utils.py

Reply:
Done.